spec/fabricators/user_fabricator.rb

```diff
@@ -54,6 +54,8 @@
 
   after_create do |user|
     user.group_users << Fabricate(:group_user, user: user, group: Group[:moderators])
+    # HACK: Some plugins add the user to staff group already which breaks because of duplication
```

Comment:
Any reasons why some (which?) plugins are doing this? 🤔

Reply:
It is because the core was not doing it and some plugins have this "fix". I only remember discourse-assign, but I think there was another one too. For example, here:

https://github.com/discourse/discourse-assign/blob/main/spec/requests/assign_controller_spec.rb#L11

Reply:
Do we still need this hack now that the `staff` bypass was restored? 🤔